documentation/assemblies/security/assembly-security.adoc

```diff
@@ -6,7 +6,7 @@
 = Managing TLS certificates
 
 Strimzi supports encrypted communication between the Kafka and Strimzi components using the TLS protocol.
-Communication between Kafka brokers (interbroker communication), between ZooKeeper nodes (internodal communication), and between these and the Strimzi operators is always encrypted.
+Communication between Kafka brokers (inter-broker communication), between ZooKeeper nodes (inter-nodal communication), and between those components and the Strimzi operators is always encrypted.
```

Comment:
If we change these, we don't match the accompanying diagram

Reply:
Good point. I removed the hyphens.